config/management/src/lib.rs

```diff
@@ -177,77 +174,16 @@ impl Command {
     }
 
     pub fn verify(self) -> Result<String, Error> {
-        if let Command::Verify(backend) = self {
-            let storage: Box<dyn Storage> = backend.backend.try_into()?;
-            if !storage.available() {
-                return Err(Error::LocalStorageUnavailable);
-            }
-
-            let mut buffer = String::new();
-
-            writeln!(buffer, "Data stored in SecureStorage:").unwrap();
-            writeln!(buffer, "=================================================").unwrap();
-            writeln!(buffer, "Keys").unwrap();
-            writeln!(buffer, "=================================================").unwrap();
-
-            Self::write_key(storage.as_ref(), &mut buffer, CONSENSUS_KEY);
-            Self::write_key(storage.as_ref(), &mut buffer, FULLNODE_NETWORK_KEY);
-            Self::write_key(storage.as_ref(), &mut buffer, OWNER_KEY);
-            Self::write_key(storage.as_ref(), &mut buffer, OPERATOR_KEY);
-            Self::write_key(storage.as_ref(), &mut buffer, VALIDATOR_NETWORK_KEY);
-
-            writeln!(buffer, "=================================================").unwrap();
-            writeln!(buffer, "Data").unwrap();
-            writeln!(buffer, "=================================================").unwrap();
-
-            Self::write_u64(storage.as_ref(), &mut buffer, EPOCH);
-            Self::write_u64(storage.as_ref(), &mut buffer, LAST_VOTED_ROUND);
-            Self::write_u64(storage.as_ref(), &mut buffer, PREFERRED_ROUND);
-            Self::write_waypoint(storage.as_ref(), &mut buffer, WAYPOINT);
-
-            writeln!(buffer, "=================================================").unwrap();
-
-            Ok(buffer)
+        if let Command::Verify(verify) = self {
```

Comment:
Nit: I always found the name for the "Verify" command a little odd (seeing as though it doesn't actually perform any checks). Do you see the code being updated to perform verification checks? If not, "Export" might be a better name (if the entire purpose is to just print it out to the console)?

Reply:
Well this has a couple means:
1) Verify you're initialized properly if you look at the proposal, we call this NodeVerifier
2) Given timelines, I'll actually be adding an optional parameter for genesis (and maybe a remote store) to confirm correct formation